core/lib/multivm/src/versions/vm_boojum_integration/vm.rs

```diff
@@ -86,7 +86,7 @@ impl<S: WriteStorage, H: HistoryMode> VmInterface<S, H> for Vm<S, H> {
     /// This method should be used only after the batch execution.
     /// Otherwise it can panic.
     fn get_current_execution_state(&self) -> CurrentExecutionState {
-        let (deduplicated_events_logs, raw_events, l1_messages) = self.state.event_sink.flatten();
```

Comment:
I'm not sure that you have to change old vms behaviour

Reply:
Why not? The behavior is the same as for `vm_latest`, if `deduplicated_events_logs` is not needed for commitment generator then we can just remove it

Reply:
just for better compatibility. 
i'd prefer to freeze the code there as much as possible

Reply:
This means we should keep `deduplicated_events_logs` in response type forever (or at least with the current multivm approach). In general I don't mind but would prefer to remove code that is not needed

Reply:
i think, you can remove it from interface, but just don't remove the implementation